src/test/java/org/folio/edge/patron/MainVerticleTest.java

```diff
@@ -311,13 +312,69 @@ public void testGetAccountPatronFoundIncludeAll(TestContext context) throws Exce
     assertEquals(expected, actual);
   }
 
+  @Test
+  public void testGetAccountPatronFoundIncludeAllAndLimitEqualsToOne(TestContext context) throws Exception {
+    logger.info("=== Test getAccount/includeLoans,includeCharges,includeHolds & limit=1 ===");
+
+    final Response resp = RestAssured
+      .get(String.format("/patron/account/%s?apikey=%s&includeLoans=true&includeHolds=true&includeCharges=true&limit=1&offset=0",
+        patronId, apiKey))
+      .then()
+      .statusCode(200)
+      .header(HttpHeaders.CONTENT_TYPE, APPLICATION_JSON)
+      .extract()
+      .response();
+
+    String expected = PatronMockOkapi.getAccountWithSingleItemsJson(patronId, true, true, true);
+    String actual = resp.body().asString();
+
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void testGetAccountPatronFoundIncludeLoansAndSortBy(TestContext context) throws Exception {
+    logger.info("=== Test getAccount/includeLoans & sortBy ===");
+
+    final Response resp = RestAssured
+      .get(String.format("/patron/account/%s?apikey=%s&includeLoans=true&includeHolds=false&includeCharges=false&sortBy=testSort&limit=10&offset=0",
+        patronId, apiKey))
+      .then()
+      .statusCode(200)
+      .header(HttpHeaders.CONTENT_TYPE, APPLICATION_JSON)
+      .extract()
+      .response();
+
+    String expected = PatronMockOkapi.getAccountWithSortedLoans(patronId);
+    String actual = resp.body().asString();
+
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  public void testGetAccountOffsetIsWrong(TestContext context) throws Exception {
+    logger.info("=== Test getAccount offset is wrong ===");
+
+    int expectedStatusCode = 400;
+    final Response resp = RestAssured
+      .get(String.format("/patron/account/%s?apikey=%s&includeLoans=true&includeHolds=true&includeCharges=true&limit=1&offset=-1",
+        patronId, apiKey))
+      .then()
+      .statusCode(expectedStatusCode)
+      .extract()
+      .response();
+
+    ErrorMessage msg = ErrorMessage.fromJson(resp.body().asString());
+    assertEquals(wrongOffsetMessage, msg.message);
+    assertEquals(expectedStatusCode, msg.httpStatusCode);
+  }
+
   @Test
   public void testGetAccountEmptyQueryArgs(TestContext context) throws Exception {
     logger.info("=== Test getAccount with empty query args ===");
 
     final Response resp = RestAssured
       .get(
-          String.format("/patron/account/%s?apikey=%s&includeCharges=&includeLoans=&includeCharges=", patronId, apiKey))
+          String.format("/patron/account/%s?apikey=%s&includeCharges=&includeLoans=&includeCharges=&limit=10&offset=0", patronId, apiKey))
```

Comment:
Given these new parameters are intended to be optional, why do they need to be added to existing tests?

Reply:
removed optional params from all existing tests